src/mobu/handlers/external.py

```diff
@@ -104,6 +111,20 @@ async def get_flock(
     return context.manager.get_flock(flock).dump()
 
 
+@external_router.put(
+    "/flocks/{flock}",
+    responses={404: {"description": "Flock not found", "model": ErrorModel}},
+    status_code=202,
+    summary="Signal a flock to refresh",
+)
+async def refresh_flock(
+    flock: str,
+    context: Annotated[RequestContext, Depends(context_dependency)],
```

Comment:
Yeah, an empty PUT feels semantically weird to me as a way to trigger this, since I think of PUT as creating or replacing the whole resource. I think I would use POST to `/flocks/{flock}/refresh` (using the action verb REST pattern) or POST to `/flocks/{flock}` with a body of `{"refreshing": true}` (using the attribute update pattern), although I don't really like the latter because refreshing doesn't feel like an attribute.

Reply:
POST to `/flocks/{flock}/refresh` sounds good to me